client/src/views/home/HomePage.vue

```diff
@@ -268,6 +272,35 @@ async function onOrganizationSelected(device: AvailableDevice): Promise<void> {
     const handle = getDeviceHandle(device);
     switchOrganization(handle, false);
   } else {
+    if (isTrialOrganizationDevice(device) && isExpired(getDurationBeforeExpiration(device.createdOn))) {
+      const answer = await askQuestion('HomePage.expiredDevice.questionTitle', 'HomePage.expiredDevice.questionMessage', {
```

Comment:
Since closing this modal with `escape` or any click outside fires an `Answer.No` which results in login, I'd add `backdropDismiss: false` to the options